pinot-tools/src/main/java/org/apache/pinot/tools/perf/QueryRunner.java

```diff
@@ -157,6 +165,12 @@ public boolean execute()
       return false;
     }
 
+    if (_useTLS) {
+      SSLContext ctx = SSLContext.getInstance("TLS");
+      ctx.init(new KeyManager[0], new TrustManager[]{new DefaultTrustManager()}, new SecureRandom());
+      SSLContext.setDefault(ctx);
+    }
```

Comment:
this is not the right way of configuring SSLContext. 
all of these logics are currently encapsulated inside TlsUtils (see #8329). they need to be managed and acquire properly.

Reply:
instead i think what's possible is when trustManager is null in 
`TlsUtils.installDefaultSSLSocketFactory`
we pass in the default TrustManager. although I am not sure if that poses any side-effect.

Reply:
This works as temporary workaround to allow us send query to perf cluster. I tested it out and it is working well. I am not sure about how to approach this another way.
@saurabhd336 coded this version. I think it is good enough for now and we can iterate on what's the right thing to do.